tomviz/CentralWidget.cxx

```diff
@@ -146,6 +157,7 @@ void PopulateHistogram(vtkImageData *input, vtkTable *output)
   output->AddColumn(populations.GetPointer());
 }
 
+//-----------------------------------------------------------------------------
```

Comment:
I generally prefer having separators too.  I've been trying to add them the whole time on this project...